db/migrate/20201208195636_add_latest_article_updated_at_to_users_and_orgs.rb

```diff
@@ -0,0 +1,6 @@
+class AddLatestArticleUpdatedAtToUsersAndOrgs < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :latest_article_updated_at, :datetime
+    add_column :organizations, :latest_article_updated_at, :datetime
```

Comment:
1. Not my favorite column name. Taking suggestions :)
2. Not sure if I need to add anything else for this to be a safe migration.

Reply:
1. I can't think of a better column name either.
2. I think it should be okay since you don't have a default value
 
Thinking out loud here: `user.latest_article_updated_at` is going to be nil, because there's no default value. That shouldn't cause any issue though since each cache usage includes Article's ID.